### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by Vhirttuegaming** [3kh0/ext-remover#815] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hello I'm trying to complete Step 2B.3 of the Skiovox exploit but pressing "Open in settings"  does nothing... any help?

**Comment by Hilololsusy** [comment, marked as answer]
Try it again, it works fine for me (did you actually do it corectly)

**Reply by Vhirttuegaming** [reply]
Thank you, turns out I wasn't pressing the question mark at the accessibility.
